logging/cluster-logging-release-notes.adoc

```diff
@@ -8,12 +8,9 @@ toc::[]
 [id="openshift-logging-supported-versions"]
 == Supported versions
 
-* OpenShift Logging versions 5.0, 5.1, and 5.2 run on {product-title} versions 4.7 and 4.8.
+OpenShift Logging versions 5.0, 5.1, and 5.2 run on {product-title} versions 4.7 and 4.8.
 
-[id="openshift-logging-inclusive-language"]
-== Making open source more inclusive
-
-Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].
+include::modules/con_making-open-source-more-inclusive.adoc[leveloffset=+1]
```

Comment:
Hi @rolfedh this seems like a good idea. How about letting the core docs team and the ocp-aligned devtools docs team know about this, so that folks can reuse it. Adding instructions in the [conscious language section](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language) like how it is done for the [tech preview module](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features) here, would help everyone.

Reply:
Thanks for the suggestion, @Preeticp. I went ahead and got this rolling: https://issues.redhat.com/browse/RHDEVDOCS-3332